CIP-015x/README.md

```diff
@@ -0,0 +1,417 @@
+
+##### Should files be byte-identical?
+
+Current approach does not provide byte-identical files, only the domain data that is stored and it's hashes are canonical. It means that tools like Mithril will have to use additional tooling or recalculate hash on their own. It's done for the purpose, this way software may add additional metadata entries, e.g. Mithril can add its own signatures to the file without violating validation properties. Other implementation may add records that are required for them to operate or bootstrap. It's true that other [approaches](https://hackmd.io/Q9eSEMYESICI9c4siTnEfw), does solve that issue by creating multiple files, each of them will be byte-identical.
+
+There are few solutions that we propose:
+
+1. allow tooling to export (or even generate) raw cbor files, that will have required byte-identical property
+2. set additional restrictions on the policies for the records, and instead of defining variable size records require all the records to have exact number of entries inside. It will harm some properties of the hardware but the files will be byte-identical in case if they have similar metadata. Or the metadata we can place it in separate file, then everything will be byte-identical.
+
+## Open Questions
+
+**What is the exact implementation for data compression, especially for indexing and search?**
+
+There are many ways to write indices, hashtables or btrees, each of them may have interesting
+properties. It's an open question which of them do we want to support.
+
+**Do we want the file be optimized for querying with external tools? If so how to achieve that?**
+
+We are proposing adding additional records types:
+
+- bloom records — they would allow faster search of the values by the key, still require file traversal
+- index records - it would allow faster search by key without full file traversal
+
+Both changes will not change the structure of the file.
+
+**Do we want to support entries without natural keys? If so how can we do that?**
+
+There are three options that we see:
+
+- In chunks records make key optional, that will support such values. That will change the spec, as we must allow many of such values in the chunks records.
+- Keep value in key, and zero value.
+- Create a separate record type for values without keys.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] Expert review and consensus from Ledger Committee, IOG, and Node teams.
+- [ ] Reference implementation in Cardano node with CLI tool for export/import.
+- [ ] Verified test vectors showing identical output across implementations, including Mithril compatibility.
+- [ ] Full documentation and CDDL schemas.
+
+### Implementation Plan
+
+1. [ ] Prototype SCLS writer/reader.
+1. [ ] Refine specification and finalise CDDL.
```

Comment:
```suggestion
2. [ ] Refine specification and finalise CDDL.
```
fix list :)

Reply:
this syntax is acceptable on GitHub... you can use any number you want 🧐